docs/Bundle-Lifecycle.md

```diff
@@ -0,0 +1,48 @@
+## Lifecycle of a Bundle
+
+<img src='./images/codalab-bundle-states.png' />
+
+<br />
+
+There are three different bundle types: `run` bundles, `uploaded` bundles and `make`
```

Comment:
There are three types of bundles: `uploaded` bundles, `make` bundles, and `run` bundles.